application-templates/javascript/event/src/connector/post-deploy.js

```diff
@@ -3,22 +3,46 @@ dotenv.config();
 
 import { createApiRoot } from '../client/create.client.js';
 import { assertError, assertString } from '../utils/assert.utils.js';
-import { createCustomerCreateSubscription } from './actions.js';
+import {
+  createGcpPubSubCustomerCreateSubscription,
+  createAzureServiceBusCustomerCreateSubscription,
+} from './actions.js';
 
 const CONNECT_GCP_TOPIC_NAME_KEY = 'CONNECT_GCP_TOPIC_NAME';
 const CONNECT_GCP_PROJECT_ID_KEY = 'CONNECT_GCP_PROJECT_ID';
+const CONNECT_PROVIDER_KEY = 'CONNECT_PROVIDER';
```

Comment:
Nit: `CONNECT_PROVIDER` to `CTP_REGION`

Reply:
I think this ties to connect provider as it determines which cloud provider we are deploying to and following the convention, CONNECT prefix seems to align more then CTP prefix.

 Should we keep it shorter as CONNECT_PROVIDER instead of CONNECT_CLOUD_PROVIDER?